src/Organizer/Events/TitleTranslated.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace CultuurNet\UDB3\Organizer\Events;
+
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Title;
+
+class TitleTranslated extends TitleUpdated
```

Comment:
Technisch gezien correct, maar voegt misschien te veel complexiteit toe voor wat je aan herbruik wint? Zeker in het kader van domain events.